environments/common.yaml.gotmpl

```diff
@@ -79,10 +105,84 @@ postgresqlHA:
     database: api
     repmgrPassword: such-secret
 
+qkm:
+  enabled: true
+  url: "http://quorum-key-manager-quorumkeymanager:8080"
+  proto: https
+  fullname: "quorumkeymanager"
+  orchestrate:
+    storeName: "eth-accounts"
+    apiKey: {{ env "QKM_API_KEY" | default "NmwyMCB3YXMgaGVyZQ==" }}
+  port: 8080
+  namespace: {{ env "QKM_NAMESPACE" | default $defaultNamespace  }}
+  chart:
+    name: {{ env "QKM_CHART" | default $defaultQkmChart  }}
+    version: {{ env "QKM_CHART_VERSION" | default $defaultQkmChartVersion  }}
+
+  image:
+    repository: {{ env "QKM_REPOSITORY" | default "docker.io/consensys/quorum-key-manager" }}
+    tag: {{ env "QKM_TAG" | default $qkmTag | quote }}
+    pullPolicy: Always
+
+  b64Manifests: {{ requiredEnv "B64_MANIFESTS" }}
+
+  environment: {}
+  environmentSecrets: {}
+
+  postgresql:
+    tls:
+      enabled: false
+    host: {{ env "QKM_POSTGRES_HOST" | default "qkm-postgresql" }}
+    port: 5432
+    database: {{ env "QKM_POSTGRES_DB" | default "qkmDB" }}
+    username: {{ env "QKM_POSTGRES_USER" | default "qkm" }}
+    password: {{ env "QKM_POSTGRES_PWD" | default "qkmDBPwd" }}
```

Comment:
Why do you have default pass in plain text?

Reply:
For the sake of simplicity it enables end users to have a demo solution running faster, and there again as stated in the README these values are under end user responsibility.